live-examples/css-examples/multi-column-layout/columns.html

```diff
@@ -0,0 +1,35 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="columns">
+<div class="example-choice">
+<pre><code id="example_one" class="language-css">columns: 2;</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
+<div class="example-choice">
+<pre><code id="example_two" class="language-css">columns: 10rem 2;</code></pre>
```

Comment:
I wonder, could something like this be interesting here? `columns: 6rem auto;` - When you play around with the `rem` value you can see how the browser splits the content into more or less rows. Thoughts @mfluehr

Reply:
Yes, good idea.